### ISSUE TITLE
check-es-heap fails with 400 in Elasticsearch 5.0.0

### ISSUE THREAD

**Issue by christianherro** [sensu-plugins/sensu-plugins-elasticsearch#53]
Starting a new cluster with Elasticsearch 5.0.0 and existing working sensu checks produces the output below when running check-es-heap.

```
/opt/sensu/embedded/bin/check-es-heap.rb
Check failed to run: 400 Bad Request, ["/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/abstract_response.rb:74:in `return!'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:495:in `process_result'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:421:in `block in transmit'", 
"/opt/sensu/embedded/lib/ruby/2.3.0/net/http.rb:853:in `start'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/resource.rb:51:in `get'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-elasticsearch-1.0.0/bin/check-es-heap.rb:112:in `acquire_es_resource'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-elasticsearch-1.0.0/bin/check-es-heap.rb:125:in `acquire_heap_data'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-elasticsearch-1.0.0/bin/check-es-heap.rb:143:in `run'", 
"/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugin-1.4.2/lib/sensu-plugin/cli.rb:58:in `block in <class:CLI>'"]
```

This seems to be the result of the inclusion of "?jvm=true" in lines 125 & 127. Running ```curl http://foo.bar:9200/_nodes/_local/stats?jvm=true``` throws a 400 error -- ```request [/_nodes/_local/stats] contains unrecognized parameter: [jvm]``` while removing that parameter returns properly. Testing seems to indicate that removing this parameter works in both Elasticsearch 5.0.0 and 2.3.4.

**Comment by christianherro** [comment]
Pull request created, changes are minimal, but very willing to take feedback on my first public pull request or anything I missed.

https://github.com/sensu-plugins/sensu-plugins-elasticsearch/pull/54

**Comment by christianherro** [comment]
Closed, pull request was merged.

**Comment by eheydrick** [comment]
Fix released in 1.1.0, which is now live on rubygems.
